mailpoet/lib/Config/ServicesChecker.php

```diff
@@ -25,6 +25,10 @@ public function __construct() {
     $this->subscribersFeature = ContainerWrapper::getInstance()->get(SubscribersFeature::class);
   }
 
+  public function isPremiumPluginActive() {
```

Comment:
I am not sure about this method. It is called `isPremiumPluginActive` but it doesn't check if the premium plugin is active, it checks the license. So would it make sense to use a different name? Why do we even need this method? I think we already check about the existence of the premium plugin, don't we?

Reply:
@pavel-mailpoet  From what I could see in the codebase, the way we are checking if the Premium plugin is active is by checking for the existence of the `MAILPOET_PREMIUM_LICENSE` constant, done with `License::getLicense()`. I'm trying to extract that out, so we won't keep using `License::getLicense()`

Reply:
Oh, right, I thought the License is for the API key. That makes sense. Good job creating the method then :)